FirebaseRemoteConfig/Sources/RCNConfigContent.m

```diff
@@ -428,7 +428,8 @@ - (FIRRemoteConfigUpdate *)getConfigUpdateForNamespace:(NSString *)FIRNamespace
 
   // add params with new/updated p13n metadata
   for (NSString *key in [fetchedP13n allKeys]) {
-    if (activeP13n[key] == nil || ![activeP13n[key] isEqualToString:fetchedP13n[key]]) {
+    if (activeP13n[key] == nil || ([fetchedP13n[key] isKindOfClass:[NSString class]] &&
```

Comment:
I think the bug here is that P13n metadata is stored as a `NSDictionary` rather a string. Just merged in a fix for that https://github.com/firebase/firebase-ios-sdk/pull/10756

Reply:
Nice!!